Resources/Prototypes/_Nuclear14/Entities/Markers/Spawners/mobs.yml

```diff
@@ -1,4 +1,4 @@
-# Passive Animals
+#MARK: Passive Animals
```

Comment:
What is the "MARK:" change on this and subsequent lines for?

Reply:
It just shows up on the side when you are scrolling through the yaml. It's to make things easier to find. Shouldn't change the code

Reply:
It's something the yaml plugin does for VSCode I'm told.